Objects/TI4_Helpers/TI4_UnitHelper.ttslua

```diff
@@ -920,6 +944,27 @@ local _unitModifiers = {
         end
     },
 
+    ['Ta Zern'] = {
+        description = 'You may reroll any dice. (Automatic tool: Reroll all misses)',
```

Comment:
Don't need the parenthetical phrase as this works even for manual use, no?

Reply:
The Commander actually let's you reroll any die, including rerolling hits. So if you negotiate with someone, maybe they reroll only their hits. Can't know the player's intention though, so for this case I figure the auto roller should just assume you're trying for maximum damage.

Reply:
Gotcha.  Maybe spell it out a bit more: "(When active will reroll all misses)".